awx/main/analytics/broadcast_websocket.py

```diff
@@ -20,7 +20,7 @@
 BROADCAST_WEBSOCKET_REDIS_KEY_NAME = 'broadcast_websocket_stats'
 
 
-logger = logging.getLogger('awx.analytics.broadcast_websocket')
+logger = logging.getLogger('awx.main.analytics.broadcast_websocket')
```

Comment:
This appears to have been a mistake. There's only 1 log that uses this, and it's fairly incoherent to sell as one of the loggers for external logging. Clearly, it wasn't intended. This change puts it back in its place as a normal system log.